app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -328,7 +328,8 @@ private void parseSusiResponse(SusiResponse susiResponse) {
 
     private void getOldMessages() {
         if (isNetworkConnected()) {
-            Call<MemoryResponse> call = clientBuilder.getSusiApi().getChatHistory();
+            ClientBuilder clientBuilder1 = new ClientBuilder();
+            Call<MemoryResponse> call = clientBuilder1.getSusiApi().getChatHistory();
```

Comment:
Yes I was previously using that but it could not be initialized and getaoldMessage gets called before it. Thus it was giving null pointer exception.